packages/dmn-js-shared/test/spec/base/ManagerSpec.js

```diff
@@ -297,17 +370,19 @@ describe('Manager', function() {
             manager._updateViews();
 
             // then
-            expect(viewsChangedSpy).to.have.been.called;
+            expect(viewsChangedSpy).to.not.have.been.called;
```

Comment:
Is this a behavior change related to the promisification? Or was it a bug in the existing test suite?

Reply:
This was a bug in the test suite if I am not mistaken.

I realized it as part of a clean-up when ensuring `done` is called when using callbacks. 
Previously the `done` was not included, hence the test would never had fail.

Reply:
Cool stuff :+1: